packages/scripts/internationalization/scripts/extract.js

```diff
@@ -61,6 +61,22 @@ function extractFiles({ files, target }) {
 	});
 }
 
+function extractByExpression({ expression, rootPath, target }) {
+	const child = spawn.sync('find', [rootPath, '-iname', `${expression}`]);
```

Comment:
does find exists on windows ?

Reply:
Nope, but it's not the only shell command that is done. I used the simpler way (shell or node) depending on the need.
The scripts will be executed in CI on linux distributions, I chose the easiest way to write it, no windows compatibility